src/playground.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <string>
+#include <filesystem>
+#include <vector>
+#include <fstream>
+#include <bits/stdc++.h>
```

Comment:
never ever use this header (it makes your code unportable i.e. it works only on gcc)
and it is useful to learn were (in which header)  each functionality lives